doc/how_to_guides/pick_ik/config/kinematics_pick_ik.yaml

```diff
@@ -0,0 +1,13 @@
+
+# Configure pick_ik as the solver to use with panda_arm
+panda_arm:
+  kinematics_solver: pick_ik/PickIkPlugin # The IK solver plugin to use
+  kinematics_solver_timeout: 0.05 # Maximum duration in seconds the solver is allowed to run per attempt
+  kinematics_solver_attempts: 3 # Maximum number of solver attempts to find a solution
+  mode: global # Solver mode 'global' enables evolutionary algorithm with gradient descent, and 'local' uses only gradient descent
+  rotation_scale: 0.5 # A scale factor for the orientation cost, setting this to 0.0 means position only IK
+  position_threshold: 0.001 # Threshold for pose difference in meters to consider a solutoin valid
+  orientation_threshold: 0.01 # Threshold for orientation difference in radians to consider a solution valid
+  cost_threshold: 0.001 # Maximum cost function value for a valid solution (all cost functions must return a value lower than this)
+  minimal_displacement_weight: 0.0 # Weight for joint angle difference cost function. Leave it low for far goals, and make higher for interpolation or jogging
```

Comment:
I would actually recommend to leave it at zero (not just low) for far goals. Maybe you can say "low or zero"?

Reply:
Sure, thanks for pointing this out! It is modified as you suggested.